                                      Summary Statistics



In [17]:
import pandas as pd
disney_data = pd.read_csv('disney movie total gross.csv')

# Clean data types for analysis
disney_data['Date Released'] = pd.to_datetime(disney_data['Date Released'], dayfirst=True, errors='coerce')
disney_data['Year Released'] = disney_data['Date Released'].dt.year


# Generate summary statistics for the dataset
disney_data.describe()


Date Released   Total Gross  Inflation Adjusted Gross  \
count                            579  5.790000e+02              5.790000e+02   
mean   1998-07-10 13:20:49.740932608  6.470179e+07              1.187625e+08   
min              1937-12-21 00:00:00  0.000000e+00              0.000000e+00   
25%              1993-03-29 12:00:00  1.278886e+07              2.274123e+07   
50%              1998-09-04 00:00:00  3.070245e+07              5.515978e+07   
75%              2006-02-06 12:00:00  7.570903e+07              1.192020e+08   
max              2016-12-16 00:00:00  9.366622e+08              5.228953e+09   
std                              NaN  9.301301e+07              2.860853e+08   

       Year Released  
count     579.000000  
mean     1998.012090  
min      1937.000000  
25%      1993.000000  
50%      1998.000000  
75%      2006.000000  
max      2016.000000  
std        11.410924

Here from the summary statistics of the Disney dataset, we observed that it contains 579 movies released between 1937 and 2016. The dataset has 13 unique genres, with Comedy being the most frequent, appearing 182 times, and PG as the most common MPAA rating with 187 movies. The financial data shows that total gross revenue ranges from 0 to 936.6 million USD, with a mean of 64.7 million USD. After After adjusting for inflation, the maximum   gross is 5.2 billion USD, indicating the immense value of Disney’s movies over time.


In [19]:
# Analyzing total gross and inflation-adjusted gross by genre
gross_by_genre = disney_data.groupby('Genre').agg({
    'Total Gross': ['mean', 'median', 'sum', 'count'],
    'Inflation Adjusted Gross': ['mean', 'median', 'sum', 'count']
}).sort_values(by=('Total Gross', 'sum'), ascending=False)

print("\nDisney Movie Summary Statistics by Genre:")
gross_by_genre


Disney Movie Summary Statistics by Genre:


Total Gross                                 \
                             mean      median          sum count   
Genre                                                              
Adventure            1.270471e+08  65000000.0  16389069453   129   
Comedy               4.461329e+07  26923500.5   8119619678   182   
Action               1.046141e+08  43315244.5   4184563282    40   
Drama                3.602608e+07  19730097.0   4106972970   114   
Thriller/Suspense    5.861694e+07  28153827.0   1406806519    24   
Musical              7.233026e+07  53041787.5   1157284155    16   
Romantic Comedy      5.009595e+07  32680633.0   1152206855    23   
Western              5.128735e+07  56932305.0    359011459     7   
Documentary          1.129285e+07   9688331.0    180685619    16   
Unknown              6.943942e+06   3218882.0    118047013    17   
Concert/Performance  5.172823e+07  51728233.0    103456466     2   
Black Comedy         3.251440e+07  28084357.0     97543212     3   
Horror               1.451148e+07  13439137.5     87068872     6   

                    Inflation Adjusted Gross                                  
                                        mean       median          sum count  
Genre                                                                         
Adventure                       1.903974e+08  102254492.0  24561266158   129  
Comedy                          8.466773e+07   51195217.0  15409526913   182  
Action                          1.374734e+08   69012064.5   5498936786    40  
Drama                           7.189302e+07   39325794.0   8195804484   114  
Thriller/Suspense               8.965379e+07   51804588.0   2151690954    24  
Musical                         6.035979e+08  103167587.5   9657565776    16  
Romantic Comedy                 7.777708e+07   57925202.0   1788872933    23  
Western                         7.381571e+07   89042541.0    516709946     7  
Documentary                     1.271803e+07   12344483.5    203488418    16  
Unknown                         2.162373e+07    6654819.0    367603384    17  
Concert/Performance             5.741084e+07   57410839.0    114821678     2  
Black Comedy                    5.224349e+07   51579764.0    156730475     3  
Horror                          2.341385e+07   18589709.5    140483092     6

From the analysis of total gross and inflation-adjusted gross by genre, we observed that Adventure and Comedy are the most financially successful genres. Adventure movies, despite having fewer releases (129), lead in total gross with a sum of 1.64 billion USD and the highest inflation-adjusted gross sum of 24.5 billion USD . Comedy movies dominate in count (182) and show consistent financial success, with a total gross sum of 811 million USD . Drama, Thriller/Suspense, and Musical genres also contribute significantly to overall revenue but with fewer releases. These insights emphasize the financial importance of focusing on Adventure and Comedy films, which consistently deliver high returns for Disney.

In [21]:
# Analyzing total gross by MPAA Rating
gross_by_rating = disney_data.groupby('MPAA Rating').agg({
    'Total Gross': ['mean', 'median', 'sum', 'count'],
    'Inflation Adjusted Gross': ['mean', 'median', 'sum', 'count']
}).sort_values(by=('Total Gross', 'sum'), ascending=False)

print("\nDisney Movie Summary Statistics by MPAA Rating:")
gross_by_rating


Disney Movie Summary Statistics by MPAA Rating:


Total Gross                                 \
                     mean      median          sum count   
MPAA Rating                                                
PG           7.362521e+07  50047179.0  13767914371   187   
PG-13        8.118074e+07  32680633.0  11771207588   145   
G            9.209061e+07  65140890.5   7919792693    86   
R            2.936536e+07  19168928.5   2995266596   102   
Unknown      1.529940e+07  12967996.0    856766536    56   
Not Rated    5.046259e+07   9230769.0    151387769     3   

            Inflation Adjusted Gross                                  
                                mean       median          sum count  
MPAA Rating                                                           
PG                      1.015414e+08   69055550.0  18988248082   187  
PG-13                   1.029486e+08   51373912.0  14927544680   145  
G                       2.912610e+08  103154765.5  25048445571    86  
R                       5.530581e+07   31587673.5   5641192166   102  
Unknown                 5.818661e+07   32106907.0   3258450260    56  
Not Rated               2.998734e+08  109581646.0    899620238     3


From the analysis of total gross and inflation-adjusted gross by MPAA rating,  
we found that PG-rated movies dominate both in frequency 187 movies and total gross, with a cumulative revenue of 1.37 billion USD.

 G-rated films,although fewer in number 86 movies , exhibit the highest median total gross of 65.1 million USD and significant inflation-adjusted revenue, with a total of 25 billion USD, PG-13 movies also perform well, contributing 1.17 billion USD to total gross and showing consistent financial success.

 R-rated films and movies with unknown ratings contribute less, highlighting Disney’s focus on family-friendly content as a key driver of success. These findings reinforce the importance of PG and G-rated movies  for maximizing revenue while maintaining the company’s brand identity.  


In [12]:
# Analyze number of movies released by decade
disney_data['Decade'] = (disney_data['Year Released'] // 10) * 10
movies_by_decade = disney_data.groupby('Decade').size()
print("\nDisney Movie Release Count by Decade:")
print(movies_by_decade)


Disney Movie Release Count by Decade:
Decade
1930      1
1940      3
1950      4
1960      7
1970      9
1980     61
1990    236
2000    172
2010     86
dtype: int64


Here, we observed that the volume of releases increased significantly over time. In the 1930s, there was just 1 movie, which grew modestly through the 1940s (3 movies) and 1950s (4 movies). A notable increase occurred in the 1980s with 61 movies, peaking in the 1990s with an impressive 236 movies. This was followed by 172 releases in the 2000s and a slight decline to 86 movies in the 2010s. We believe this decline in the 2010s was due to Disney shifting its focus from quantity to quality, prioritizing high-budget blockbuster films from franchises like Marvel and Star Wars. This strategy ensured fewer releases but higher profitability per movie, aligning with changing audience preferences for big cinematic events.



In [22]:
# Summary of inflation-adjusted gross trends over time
gross_trends = disney_data.groupby('Year Released').agg({
    'Inflation Adjusted Gross': 'sum'
})
print("\nInflation Adjusted Gross Trends Over Time:")
gross_trends


Inflation Adjusted Gross Trends Over Time:


Inflation Adjusted Gross
Year Released                          
1937                         5228953251
1940                         4375319860
1946                         1078510579
1950                          920608730
1954                          528279994
1955                         1236035515
1959                           21505832
1961                         1797806719
1962                          109581646
1963                          153870834
1967                          789612346
1968                          138612686
1970                          356362241
1971                           91305448
1975                          131246872
1977                          363659174
1979                          120377374
1980                           43560212
1981                          181636869
1982                          110731153
1983                          100549534
1984                          178132661
1985                          173124212
1986                          577472408
1987                         1186649969
1988                          986841867
1989                         1097851241
1990                         1226460745
1991                         1221190032
1992                         1736349861
1993                         1427312980
1994                         2140690807
1995                         2188599043
1996                         2157238999
1997                         1474848034
1998                         2189031209
1999                         1981582799
2000                         1711675892
2001                         1296291781
2002                         1680599676
2003                         2171244827
2004                         1570219160
2005                         1273982639
2006                         1834744763
2007                         1757479739
2008                         1135356255
2009                         1359908617
2010                         1622535159
2011                         1172578942
2012                         1537669134
2013                         1887863558
2014                         1560791318
2015                         2495662696
2016                         2873393105

From the analysis of inflation-adjusted gross trends over time, we observed a
consistent growth in revenues across decades. Early years like 1937 saw an
inflation-adjusted gross of 5.22 billion USD, and the trend continued with
significant increases over time. The 1980s and 1990s marked substantial
growth, with peaks in the 2010s, culminating in a record-breaking
28.7 billion USD in 2016. This progression highlights Disney’s ability to
capitalize on modern blockbusters and adapt to evolving audience demands.
The consistent growth reflects both strong content strategies and the global
expansion of Disney’s brand over time.



**OVERALL CONCLUSION**

The Disney movie dataset, spanning 579 films from 1937 to 2016, highlights the company's strengths and financial successes. Comedy emerges as the most frequent and lucrative genre, while Adventure films, despite fewer releases, boast the highest median gross, reflecting consistent profitability. PG rated films dominate in both volume (187 movies) and total revenue, while G rated films, though less frequent, are incredibly profitable, with a median inflation adjusted gross of $2.9 billion. Disney’s family friendly focus is further emphasized by the minimal contribution of R rated films. Production trends show a peak in the 1990s with 236 releases, while inflation-adjusted revenues have grown over time, with modern blockbusters like those in 2016 achieving record-breaking success. Focusing on Adventure and Comedy films, alongside PG and G rated content, aligns with Disney’s brand and maximizes profitability. However, limitations in data, such as gaps in genres and MPAA ratings and skewed inflation adjustments, suggest that incorporating audience demographics and market preferences could provide deeper insights for strategic decisions.
